pages/linux/settings.md

```diff
@@ -0,0 +1,19 @@
+# settings
+
+> Get information about the Android OS.
+
+- Display a list of settings in the {{global}} namespace:
+
+`settings list global`
+
+- Get the value of a specific setting:
+
+`settings get {{global}} {{airplane_mode_on}}`
+
+- Set the value of a setting:
+
+`settings put {{system}} {{screen_brightness}} {{42}}`
```

Comment:
Here you're using `system`, but everywhere else you use `global`, which _could_ be a little confusing.  Could you use something else, or do you have stronger feelings about it?

Reply:
I wanted to show that `global` is not the only namespace available. What would be the best way to do that?

Reply:
Good idea. In that case, I'd keep this example.  How about you change the last example to use a completely different namespace, then (assuming that `system` and `global` aren't the only ones.)